config/webhook/manifests.yaml

```diff
@@ -1,5 +1,30 @@
 ---
 apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: mutating-webhook-configuration
+webhooks:
```

Comment:
Additionally to [Vedant's comment](https://github.com/opendatahub-io/odh-model-controller/pull/221#pullrequestreview-2132997987), if possible, configure some selector/filter on the webhook, to prevent it being called on every Svc on the cluster.

Reply:
+1

Reply:
at the moment, the service does not have any labels so it is not possible to filter. In order to do this, I need to send another pr to upstream kserve. I will follow up this with another ticket: [RHOAIENG-8978](https://issues.redhat.com/browse/RHOAIENG-8978)

Reply:
Remove mutatingwebhook so it is not needed